### DISCUSSION TITLE
support for EXT_meshopt_compression

### DISCUSSION THREAD

**Discussion by daktor** [assimp/assimp#5750] [Q&A]
Hi all,

Does assimp support EXT_meshopt_compression?

I have a glb with that extension.  It has the following in its header:
```
"buffers":[
{"byteLength":156516},
{"byteLength":202572,"extensions":{"EXT_meshopt_compression":{"fallback":true}}}
],

```
This causes DeadlyImportException:  "GLTF: buffer with non-zero length missing the \"uri\" attribute"

Any ideas how to fix?

Thanks!

**Comment by kimkulling** [comment]
Hi,
this extension is currently not supporte. If you think that it would be a win for the Asset-Importer-Lib you can create a feature request for it. But I have to say caused by my taff schedule during my freetime I will currently not be able to fulfill this extension. 

It would be really great if you could help us here!

Kim

**Comment by tellypresence** [comment]
Requested in #6181
